public/docs/ts/latest/guide/index.jade

```diff
@@ -120,10 +113,10 @@ block example-links
 
   # Feedback
 
-  We welcome feedback! Leave a comment by clicking the icon in upper right corner of the banner.
+  We welcome feedback! To leave a comment click the icon in the upper right corner of the banner.
 
-  Post *documentation* issues and pull requests on the
+  Post documentation issues and pull requests on the
   [angular.io](https://github.com/angular/angular.io) github repository.
 
-  Post issues with *Angular 2 itself* to the [angular](https://github.com/angular/angular) github repository.
+  Post issues with *Angular 2* itself to the [angular](https://github.com/angular/angular) github repository.
```

Comment:
Maybe:

Use the [angular](...) GitHub repo to report issues with **Angular 2** itself.